renku/cli/__init__.py

```diff
@@ -173,6 +186,18 @@ def print_global_config_path(ctx, param, value):
 @click.pass_context
 def cli(ctx, path, renku_home, use_external_storage):
     """Check common Renku commands used in various situations."""
+    renku_path = Path(path) / renku_home
+
+    if not renku_path.exists() and not is_allowed_command(ctx):
+        click.echo(
+            WARNING +
+            '`{0}` is not a renku initialized repository.'.format(path)
+        )
+        raise UsageError((
```

Comment:
If you want to word it like this, you should write `renku-managed`, but I am ok with just calling it a renku repository
```'To initialize this as a renku repository, use: `renku init`'```

The use of warning and error looks a bit strange:

```
Warning: `...` is not a renku initialized repository.
Error: To initialize this as a renku managed repository use: `renku init`
```

Could this not be made to 

```
Error: `...` is not a renku repository.
To initialize this as a renku repository, use: `renku init`
```